next.clayui.com/content/docs/css/components/forms/text-input.md

```diff
@@ -0,0 +1,252 @@
+</div>
+```
+
+## Textarea
+
+<div class="sheet-example">
+	<div class="form-group">
+		<label for="basicInputTypeTextarea">Textarea</label>
+		<textarea class="form-control" id="basicInputTypeTextarea" placeholder="Placeholder"></textarea>
+	</div>
+</div>
+
+```html
+<div class="form-group">
+	<label for="basicInputTypeTextarea">Textarea</label>
+	<textarea class="form-control" id="basicInputTypeTextarea" placeholder="Placeholder"></textarea>
+</div>
+```
+
+## Validations
+
+### Success
+
+<div class="sheet-example">
+	<div class="form-group has-success">
+		<label for="inputSuccess1">
+			has-success
+			<svg class="lexicon-icon lexicon-icon-asterisk reference-mark" focusable="false" role="presentation">
+				<use href="/images/icons/icons.svg#asterisk" />
+			</svg>
+		</label>
+		<input class="form-control" id="inputSuccess1" type="text"/>
+		<div class="form-feedback-group">
+			<div class="form-feedback-item">This is a form-feedback-item.</div>
+			<div class="form-feedback-item">
+				<span class="form-feedback-indicator">
+					<svg class="lexicon-icon lexicon-icon-check-circle-full" focusable="false" role="presentation">
+						<use href="/images/icons/icons.svg#check-circle-full" />
+					</svg>
+				</span>
+				This is a form-feedback-indicator.
+			</div>
+			<div class="form-text">This is form-text.</div>
+		</div>
+	</div>
+</div>
+
+```html
+<div class="form-group has-success">
+	<label for="inputSuccess1">
+		has-success
+		<svg class="lexicon-icon lexicon-icon-asterisk reference-mark" focusable="false" role="presentation">
+			<use href="/images/icons/icons.svg#asterisk" />
+		</svg>
+	</label>
+	<input class="form-control" id="inputSuccess1" type="text"/>
+	<div class="form-feedback-group">
+		<div class="form-feedback-item">This is a form-feedback-item.</div>
+		<div class="form-feedback-item">
+			<span class="form-feedback-indicator">
+				<svg class="lexicon-icon lexicon-icon-check-circle-full" focusable="false" role="presentation">
+					<use href="/images/icons/icons.svg#check-circle-full" />
+				</svg>
+			</span>
+			This is a form-feedback-indicator.
+		</div>
+		<div class="form-text">This is form-text.</div>
+	</div>
+</div>
+```
+
+### Warning
+
+<div class="sheet-example">
+	<div class="form-group has-warning">
+		<label for="inputWarning1">
+			has-warning
+			<svg class="lexicon-icon lexicon-icon-asterisk reference-mark" focusable="false" role="presentation">
+				<use href="/images/icons/icons.svg#asterisk" />
+			</svg>
+		</label>
+		<input class="form-control" id="inputWarning1" type="text"/>
+		<div class="form-feedback-group">
+			<div class="form-feedback-item">This is a form-feedback-item.</div>
+			<div class="form-feedback-item">
+				<span class="form-feedback-indicator">
+					<svg class="lexicon-icon lexicon-icon-warning-full" focusable="false" role="presentation">
+						`<use href="/images/icons/icons.svg#warning-full" />
```

Comment:
should this backtick be here?

Reply:
Oh no, I ended up putting it on without meaning to. Thanks for catch